apps/frontend/src/renderer/components/Terminal.tsx

```
@@ -238,17 +313,28 @@ Please confirm you're ready by saying: I'm ready to work on ${selectedTask.title
         'flex h-full flex-col rounded-lg border bg-[#0B0B0F] overflow-hidden transition-all relative',
+        // Default border states
         isActive ? 'border-primary ring-1 ring-primary/20' : 'border-border',
-        isOver && 'ring-2 ring-info border-info'
+        // File drop overlay
+        showFileDropOverlay && 'ring-2 ring-info border-info',
+        // Claude busy state indicator (subtle colored border when in Claude mode)
+        showClaudeBusyIndicator && isClaudeBusy && 'border-red-500/60 ring-1 ring-red-500/20',
+        showClaudeBusyIndicator && !isClaudeBusy && 'border-green-500/60 ring-1 ring-green-500/20'
       )}
       onClick={handleClick}
     >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)